fluxcd/assets/dashboards/fluxcd.json

```diff
@@ -1,6 +1,6 @@
 {
     "author_name": "Datadog",
-    "description": "**This Dashboard provides Flux CI/CD Observability metrics for your Kubernetes clusters with the Flux Integration**\n\n# Flux Documentation\n- [Flux Integration&nbsp;\u2197](https://docs.datadoghq.com/integrations/fluxcd)\n- [Flux Metrics&nbsp;\u2197](https://docs.datadoghq.com/integrations/fluxcd/#metrics)\n- [Flux Documentation&nbsp;\u2197](https://fluxcd.io/flux/#flux-in-short)",
+    "description": "This dashboard provides an overview of your Flux CD instance. It monitors the performance of your continuous delivery and deployment processes for your containerized applications.\n\n### Useful Links\n- [Flux CD Integration&nbsp;\u2197](https://docs.datadoghq.com/integrations/fluxcd)\n- [Blog on Flux CD and other Container-native Integrations&nbsp;\u2197](https://www.datadoghq.com/blog/container-native-integrations/#cicd-with-flux)\n- [Flux CD Metrics&nbsp;\u2197](https://docs.datadoghq.com/integrations/fluxcd/#metrics)\n- [Flux CD Documentation&nbsp;\u2197](https://fluxcd.io/flux/#flux-in-short) (cloned)",
```

Comment:
```suggestion
    "description": "This dashboard provides an overview of your Flux CD instance. It monitors the performance of your continuous delivery and deployment processes for your containerized applications.\n\n### Useful Links\n- [Flux CD Integration&nbsp;\u2197](https://docs.datadoghq.com/integrations/fluxcd)\n- [Blog on Flux CD and other Container-native Integrations&nbsp;\u2197](https://www.datadoghq.com/blog/container-native-integrations/#cicd-with-flux)\n- [Flux CD Metrics&nbsp;\u2197](https://docs.datadoghq.com/integrations/fluxcd/#metrics)\n- [Flux CD Documentation&nbsp;\u2197](https://fluxcd.io/flux/#flux-in-short)",
```

Reply:
I'm not sure if it's intentional to put `cloned` here, so feel free to ignore this suggestion if so.

Reply:
very good catch, thanks!